src/cascadia/TerminalApp/Profile.cpp

```diff
@@ -494,15 +496,16 @@ bool Profile::HasIcon() const noexcept
 // - tabTitle: the tab title
 void Profile::SetTabTitle(std::wstring tabTitle) noexcept
 {
-    _tabTitle = tabTitle;
+    _tabTitle = std::move(tabTitle);
 }
 
 // Method Description:
 // - Sets this profile's icon path.
 // Arguments:
 // - path: the path
-void Profile::SetIconPath(std::wstring_view path) noexcept
+void Profile::SetIconPath(std::wstring_view path)

```

Comment:
Curious here. This one uses `wstring_view`, and the others use `wstring`. Should we standardize on just one? If so, what do you think we should do, @yves-dolce?

Reply:
If the body of the method (e.g. `Profile::SetIconPath`) calls methods that support `std::wstring_view`, using `std::wstring_view` will allow you not to have any heap allocation during the parameter passing. That class is just a pointer and a size after all.
But keep in mind that:
- it does not enforce the string to be null terminated. Again, it's just a pointer to the beginning of the string and a length. 
- the compiler might have a hard time optimizing the calling site as now, you've created an alias. When you were taking a `std;:wstring`, no aliasing was created.

Reply:
Fair enough! :+1: